R/Rules-methods.R

```diff
@@ -1725,6 +1730,15 @@ setMethod("stopTrial",
       ## so can we stop?
       doStop <- nCohorts >= stopping@nCohorts
 
+      report_label <- if (isTRUE(stopping@report_label == "")) {
```

Comment:
I think this looks better:
```suggestion
      report_label <- if (isTRUE(nzchar(stopping@report_label))) {
```
and in all remaining places the same change